src/node/interfaces.cpp

```diff
@@ -678,11 +678,11 @@ class ChainImpl : public Chain
         // that Chain clients do not need to know about.
         return TransactionError::OK == err;
     }
-    void getTransactionAncestry(const Txid& txid, size_t& ancestors, size_t& descendants, size_t* ancestorsize, CAmount* ancestorfees) override
+    void getTransactionAncestry(const Txid& txid, size_t& ancestors, size_t& cluster_count, size_t* ancestorsize, CAmount* ancestorfees) override
```

Comment:
nit if you retouch: commit message for cec26203732 should replace "descendantsize" with "descendant count"